CHANGELOG.unreleased.md

```diff
@@ -13,6 +13,10 @@ Section headings should be at level 3 (e.g. `### Added`).
 
 ## Unreleased
 
+### Added
+
+- Added an internal helper `wandb.util.one()` to make it easier to extract a single-valued iterable. (@tonyyli-wandb in https://github.com/wandb/wandb/pull/9310)
+
```

Comment:
Since this is internal, let's not add it to the changelog (which is primarily for library users rather than library developers).